### ISSUE TITLE
masquerading as audience but able to edit text object & see media that has not had lightbulb turned on

### ISSUE THREAD

**Issue by helenvarley** [upstage-org/upstage#631]
when i masquerade as audience, i am able to edit a text object that is on the stage. i logged out and checked that as real audience, i cannot edit the text.

i am also able to see a media object that has been placed on the stage but not had its lightbulb turned on - it's showing as B&W, & i can't see its frame or do anything with it, but i can still see it. 

these are minor things and definitely NOT urgent, but it could be confusing to not be seeing exactly what audience sees when we are masquerading.

![Screenshot from 2021-10-13 12:57:28](https://user-images.githubusercontent.com/3437324/137120484-6db2eae1-cdcb-4337-a8d0-b84412ec35a0.png)
![Screenshot from 2021-10-13 12:57:15](https://user-images.githubusercontent.com/3437324/137120490-f2e1f45c-eb53-464c-b7ef-9814df535ec9.png)
![masquerade](https://user-images.githubusercontent.com/3437324/137120491-1257bcc8-ccdc-4eeb-9857-eb6cf689b990.gif)

**Comment by gloriajw** [comment]
Instead of masquerading as audience, can you just have a separate browser logged in as audience?  Much more secure.

**Comment by helenvarley** [comment]
yes we could; however the masquerade button makes it easy to quickly check the audience view, & is useful especially in teaching situations to help new players understand how things are seen by the audience. it's easier to show them the masquerade button within the on-stage tools than to ask them to open a different browser (not everyone has more than one browser already installed).
but actually i think the original issue - of being able to see things i shouldn't when in masquerade mode, is actually fixed. so i'll close this issue.
(it's also not high priority)
